### ISSUE TITLE
I think there are bugs in train_pose.py

### ISSUE THREAD

**Issue by tugumi911** [last-one/Pytorch_Realtime_Multi-Person_Pose_Estimation#10]
Hi,

I ran train.sh, but I had some errors.
I think there are bugs.

Should I add "input = input.cuda()" to around line 138 in train_pose.py and change line 63 "params_dict = dict(model.module.named_parameters())" in train_pose.py to "params_dict = dict(model.named_parameters())" .

**Comment by last-one** [comment]
If you use one gpu to train and change line 49 "model = torch.nn.DataParallel(model, device_ids=args.gpu).cuda()" to "model = model.cuda()", you're right.

**Comment by tugumi911** [comment]
Thank you for your replying.

Yes, I use one gpu to train.
Should I change other codes ?

**Comment by last-one** [comment]
No, I think. If you find something error, please let me know. Thank you.

**Comment by tugumi911** [comment]
Thank you !

**Comment by xdxuefei** [comment]
@tugumi911  you are right!
if one gpu is used, "input = input.cuda()" and "params_dict = dict(model.named_parameters())"  should be modified!
thanks

**Comment by bdfish1104** [comment]
If we want to use only one GPU, there is another way can achieve it.

1. Remove the line "os.environ['CUDA_VISIBLE_DEVICES'] = '2,3'" in the train_pose.py.
2. Modify the value of --gpu in train.sh to 0.
